src/app/about/page.js

```diff
@@ -1,7 +1,31 @@
 import React from "react";
+import Mission from "@/components/about/Mission";
+import mission1 from "@/public/assets/about/mission1.svg";
+import mission2 from "@/public/assets/about/mission2.svg";
+import mission3 from "@/public/assets/about/mission3.svg";
 
 const page = () => {
-  return <div>page</div>;
+  const text1 =
```

Comment:
instead of 2 separate props for text and paragraph (I see why you did it), just have an array when each text has 2 strings. Then just map through each string. Let me know if you need help with this.

Reply:
this will reduce code a lot